frontend/components/lesson/diff/CodeDiffWithSelector.js

```diff
@@ -13,6 +13,15 @@ function selectDefaultVersions(versions) {
   return [1, 0]
 }
 
+function pad(value) {
+  return value.toString().padStart(2, '0')
+}
+
+function formatDate(dt) {
+  dt = new Date(dt)
+  return `${pad(dt.getDate())}. ${pad(dt.getMonth())}. ${pad(dt.getFullYear())} ${pad(dt.getHours())}:${pad(dt.getMinutes())}:${pad(dt.getSeconds())}`
+}
+
```

Comment:
I suggest avoiding the padding logic and using `.toLocaleString()` which best-matches the user preferred date/time format accto browser interface language, e.g. for CS it will give the most usual/official date/time format:
```
dt = new Date("2021-12-07T12:15:24Z").toLocaleString()
"7. 12. 2021 13:15:24"
```

Reply:
I have bad experiences with these Locale functions in JavaScript :D But I suppose that browsers are much better with them nowadays. Thanks, I'll change it.

Reply:
Thanks!